routes/contributions.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const router = express.Router()
+const contributionsController = require('../controllers/contributionsController')
+
+/**
+ * @swagger
+ * /contributions/:username:
```

Comment:
There's no way to add the param value in swagger docs. Update the JSDOCs to include the param values.

Reply:
Refer: https://swagger.io/docs/specification/describing-parameters/